docs/FAQ.adoc

```diff
@@ -458,3 +458,58 @@ If the server side is v2.2+, then the warning message will appear.
 Call rdiff-backup with the higher API and the message will disappear.
 
 Calling for example `rdiff-backup --api-version 201 backup loc1 server2::loc2` will make sure that `rdiff-backup server` is being called, getting rid of the warning message.
+
+== How to handle a "No space left on device" message while doing a backup?
```

Comment:
Some note of that. This error could also be raised if the "/tmp" folder is full. That happen to some one on the mailing list. I just don'T recall if this was during a backup or a restore.

Reply:
I am the opinion that we addressed the issue at some point in time, but it's not a guarantee that it can't happen so I've added a note to this regard. Thanks.